/var/lib/condor/execute/slot7/dir_90851/condor_exec.exe: line 8: lspci: command not found
  0%|          | 0/36 [00:00<?, ?it/s]Extracting : wheel-0.34.2-py37_0.conda:   0%|          | 0/36 [00:00<?, ?it/s]Extracting : wheel-0.34.2-py37_0.conda:   3%|▎         | 1/36 [00:00<00:04,  8.72it/s]Extracting : libedit-3.1.20181209-hc058e9b_0.conda:   3%|▎         | 1/36 [00:00<00:04,  8.72it/s]Extracting : libffi-3.2.1-hd88cf55_4.conda:   6%|▌         | 2/36 [00:00<00:03,  8.72it/s]        Extracting : openssl-1.1.1d-h7b6447c_4.conda:   8%|▊         | 3/36 [00:00<00:03,  8.72it/s]Extracting : openssl-1.1.1d-h7b6447c_4.conda:  11%|█         | 4/36 [00:00<00:04,  7.62it/s]Extracting : libstdcxx-ng-9.1.0-hdf63c60_0.conda:  11%|█         | 4/36 [00:00<00:04,  7.62it/s]Extracting : libstdcxx-ng-9.1.0-hdf63c60_0.conda:  14%|█▍        | 5/36 [00:00<00:05,  6.17it/s]Extracting : pycparser-2.19-py37_0.conda:  14%|█▍        | 5/36 [00:00<00:05,  6.17it/s]        Extracting : requests-2.22.0-py37_1.conda:  17%|█▋        | 6/36 [00:00<00:04,  6.17it/s]Extracting : urllib3-1.25.8-py37_0.conda:  19%|█▉        | 7/36 [00:00<00:04,  6.17it/s] Extracting : six-1.14.0-py37_0.conda:  22%|██▏       | 8/36 [00:00<00:04,  6.17it/s]    Extracting : _libgcc_mutex-0.1-main.conda:  25%|██▌       | 9/36 [00:00<00:04,  6.17it/s]Extracting : xz-5.2.4-h14c3975_4.conda:  28%|██▊       | 10/36 [00:00<00:04,  6.17it/s]  Extracting : cffi-1.14.0-py37h2e261b9_0.conda:  31%|███       | 11/36 [00:00<00:04,  6.17it/s]Extracting : ca-certificates-2020.1.1-0.conda:  33%|███▎      | 12/36 [00:00<00:03,  6.17it/s]Extracting : pip-20.0.2-py37_1.conda:  36%|███▌      | 13/36 [00:00<00:03,  6.17it/s]         Extracting : ruamel_yaml-0.15.87-py37h7b6447c_0.conda:  39%|███▉      | 14/36 [00:00<00:03,  6.17it/s]Extracting : setuptools-45.2.0-py37_0.conda:  42%|████▏     | 15/36 [00:00<00:03,  6.17it/s]          Extracting : ld_impl_linux-64-2.33.1-h53a641e_7.conda:  44%|████▍     | 16/36 [00:00<00:03,  6.17it/s]Extracting : pyopenssl-19.1.0-py37_0.conda:  47%|████▋     | 17/36 [00:00<00:03,  6.17it/s]           Extracting : conda-package-handling-1.6.0-py37h7b6447c_0.conda:  50%|█████     | 18/36 [00:00<00:02,  6.17it/s]Extracting : readline-7.0-h7b6447c_5.conda:  53%|█████▎    | 19/36 [00:00<00:02,  6.17it/s]                    Extracting : asn1crypto-1.3.0-py37_0.conda:  56%|█████▌    | 20/36 [00:00<00:02,  6.17it/s]Extracting : tqdm-4.42.1-py_0.conda:  58%|█████▊    | 21/36 [00:00<00:02,  6.17it/s]       Extracting : pycosat-0.6.3-py37h7b6447c_0.conda:  61%|██████    | 22/36 [00:00<00:02,  6.17it/s]Extracting : zlib-1.2.11-h7b6447c_3.conda:  64%|██████▍   | 23/36 [00:00<00:02,  6.17it/s]      Extracting : python-3.7.6-h0371630_2.conda:  67%|██████▋   | 24/36 [00:04<00:01,  6.17it/s]Extracting : python-3.7.6-h0371630_2.conda:  69%|██████▉   | 25/36 [00:04<00:01,  6.09it/s]Extracting : cryptography-2.8-py37h1ba5d50_0.conda:  69%|██████▉   | 25/36 [00:04<00:01,  6.09it/s]Extracting : libgcc-ng-9.1.0-hdf63c60_0.conda:  72%|███████▏  | 26/36 [00:04<00:01,  6.09it/s]     Extracting : sqlite-3.31.1-h7b6447c_0.conda:  75%|███████▌  | 27/36 [00:04<00:01,  6.09it/s]  Extracting : certifi-2019.11.28-py37_0.conda:  78%|███████▊  | 28/36 [00:04<00:01,  6.09it/s]Extracting : yaml-0.1.7-had09818_2.conda:  81%|████████  | 29/36 [00:04<00:01,  6.09it/s]    Extracting : pysocks-1.7.1-py37_0.conda:  83%|████████▎ | 30/36 [00:04<00:00,  6.09it/s] Extracting : idna-2.8-py37_0.conda:  86%|████████▌ | 31/36 [00:04<00:00,  6.09it/s]     Extracting : chardet-3.0.4-py37_1003.conda:  89%|████████▉ | 32/36 [00:04<00:00,  6.09it/s]Extracting : tk-8.6.8-hbc83047_0.conda:  92%|█████████▏| 33/36 [00:04<00:00,  6.09it/s]    Extracting : ncurses-6.2-he6710b0_0.tar.bz2:  94%|█████████▍| 34/36 [00:04<00:00,  6.09it/s]Extracting : conda-4.8.2-py37_0.tar.bz2:  97%|█████████▋| 35/36 [00:04<00:00,  6.09it/s]                                                                                            

==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.12.0

Please update conda by running

    $ conda update -n base -c defaults conda


Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/var/lib/condor/execute/slot7/dir_90851/miniconda3/envs/pytorch-gpu/lib/python3.8/site-packages/torch/cuda/__init__.py", line 276, in get_device_name
    return get_device_properties(device).name
  File "/var/lib/condor/execute/slot7/dir_90851/miniconda3/envs/pytorch-gpu/lib/python3.8/site-packages/torch/cuda/__init__.py", line 306, in get_device_properties
    _lazy_init()  # will define _get_device_properties
  File "/var/lib/condor/execute/slot7/dir_90851/miniconda3/envs/pytorch-gpu/lib/python3.8/site-packages/torch/cuda/__init__.py", line 164, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
/var/lib/condor/execute/slot7/dir_90851/condor_exec.exe: line 43: start: command not found
Traceback (most recent call last):
  File "main.py", line 167, in <module>
    main()
  File "main.py", line 120, in main
    raise OSError("Torch cannot find a cuda device")
OSError: Torch cannot find a cuda device
/var/lib/condor/execute/slot7/dir_90851/condor_exec.exe: line 48: date +%s: syntax error: operand expected (error token is "%s")
/var/lib/condor/execute/slot7/dir_90851/condor_exec.exe: line 49: runtime: command not found
